mappings/net/minecraft/util/thread/TaskExecutor.mapping

```diff
@@ -8,8 +8,11 @@ CLASS net/minecraft/class_3846 net/minecraft/util/thread/TaskExecutor
 		ARG 1 queue
 		ARG 2 executor
 		ARG 3 name
+	METHOD method_16899 (I)Z
+		ARG 0 runCount
 	METHOD method_16900 runWhile (Lit/unimi/dsi/fastutil/ints/Int2BooleanFunction;)I
 		ARG 1 condition
+			COMMENT ran task count -&gt; whether to run the next task
```

Comment:
only formatting, but an arrow? That feels weird

Reply:
Done, used regular statements instead